### PR TITLE
Add VAT ID translation

### PR DESCRIPTION
**Pull request by AK-Dev11**
#8641

### CODE REVIEW

File: packages/Webkul/Admin/src/Resources/lang/ar/app.php [hunk 1]
```diff
@@ -1396,6 +1396,7 @@
             'create-success'      => 'تم إنشاء العنوان بنجاح',
             'update-success'      => 'تم تحديث العنوان بنجاح',
             'success-mass-delete' => 'حذف العناوين بنجاح',
+            'invalid-format'      => 'تنسيق الضريبة المضافة غير صحيح',
```
Comment (amit-webkul) [inline packages/Webkul/Admin/src/Resources/lang/ar/app.php:1399]: Add your new key in alphabetical order based on the first letter of the key in all lang files.

File: packages/Webkul/Customer/src/Rules/VatIdRule.php [hunk 2]
```diff
@@ -37,6 +37,6 @@ public function passes($attribute, $value)
      */
     public function message()
     {
-        return trans('shop::app.invalid_vat_format');
+        return trans('admin::app.customers.addresses.invalid-format');
```
Comment (amit-webkul) [inline packages/Webkul/Customer/src/Rules/VatIdRule.php:40]: Please use the language from the customer package in accordance with our standard.

File: packages/Webkul/Admin/src/Resources/lang/ar/app.php (hunk 1 above)
Comment (AK-Dev11) [reply packages/Webkul/Admin/src/Resources/lang/ar/app.php]: Done.

File: packages/Webkul/Customer/src/Rules/VatIdRule.php (hunk 2 above)
Comment (AK-Dev11) [reply packages/Webkul/Customer/src/Rules/VatIdRule.php]: Done.

**Review by Amitk-webkul — APPROVED** [review]

**Review by devansh-webkul — APPROVED** [review]

File: packages/Webkul/Customer/src/Captcha.php [hunk 3]
```diff
@@ -146,8 +146,8 @@ public function getValidationMessages($messages = []): array
     {
         return $this->isActive()
             ? array_merge($messages, [
-                'g-recaptcha-response.required' => __('customer::app.admin.system.captcha.validations.required'),
-                'g-recaptcha-response.captcha' => __('customer::app.admin.system.captcha.validations.captcha')
+                'g-recaptcha-response.required' => __('customer::app.validations.captcha.required'),
+                'g-recaptcha-response.captcha' => __('customer::app.validations.captcha.captcha')
```
Comment (amit-webkul) [inline packages/Webkul/Customer/src/Captcha.php:150]: Please modify the "trans" method on the place of ( __ ).

Comment (AK-Dev11) [reply packages/Webkul/Customer/src/Captcha.php]: Replace (_) into trans method

File: packages/Webkul/Customer/src/Resources/lang/hi_IN/app.php [hunk 4]
```diff
@@ -1,14 +1,15 @@
...
+        'captcha' => [
+            'captcha'  => 'कुछ गलती हुई! कृपया पुन: प्रयास करें।',
+            'required' => 'कृपया CAPTCHA का चयन करें',
         ],
-    ],    
+
+        'vat-id' => [
+            'invalid-format' => 'अमान्य वीएट प्रारूप',
+        ],
+
```
Comment (amit-webkul) [inline packages/Webkul/Customer/src/Resources/lang/hi_IN/app.php:13]: Remove the space

Comment (AK-Dev11) [reply packages/Webkul/Customer/src/Resources/lang/hi_IN/app.php]: Done

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**